pyproject.toml

```diff
@@ -1,11 +1,61 @@
+# https://packaging.python.org/en/latest/specifications/pyproject-toml/
 [project]
 name = "spack"
 description = "The spack package manager"
+readme = "README.md"
 requires-python = ">=3.6"
-dependencies = ["clingo", "setuptools"]
+license = "MIT OR Apache-2.0"
+license-files = ["LICENSE-*"]
+authors = [
+    {name="Todd Gamblin", email="tgamblin@llnl.gov"},
+]
+maintainers = [
+    {name="Todd Gamblin", email="tgamblin@llnl.gov"},
+    {name="Adam J. Stewart", email="ajstewart426@gmail.com"},
+]
```

Comment:
Can add more people here, just let me know who we want. Maybe authors come from the first Spack paper and maintainers are a subset of people with merge privs?